cves/kernel/CVE-2014-3144.yml

```diff
@@ -1,199 +1,171 @@
-  security bulletins and bug reports. It is not necessarily the same day that
-  the CVE was created.  Leave blank if no date is given.
-
-  Please enter your date in YYYY-MM-DD format.
-reported_date:
-announced_instructions: |
-  Was there a date that this vulnerability was announced to the world? You can
-  find this in changelogs, blogs, bug reports, or perhaps the CVE date.
-
-  This is not the same as published date in the NVD - that is below.
-
-  Please enter your date in YYYY-MM-DD format.
-announced_date: '2014-05-11'
-published_instructions: |
-  Is there a published fix or patch date for this vulnerability?
-  Please enter your date in YYYY-MM-DD format.
-published_date: '2014-05-11'
-description_instructions: |
-  You can get an initial description from the CVE entry on cve.mitre.org. These
-  descriptions are a fine start, but they can be kind of jargony.
-
-  Rewrite this description IN YOUR OWN WORDS. Make it interesting and easy to
-  read to anyone with some programming experience. We can always pull up the NVD
-  description later to get more technical.
-
-  Try to still be specific in your description, but remove project-specific
-  stuff. Remove references to versions, specific filenames, and other jargon
-  that outsiders to this project would not understand. Technology like "regular
-  expressions" is fine, and security phrases like "invalid write" are fine to
-  keep too.
-
-  Your target audience is people just like you before you took any course in
-  security
-description:
-bounty_instructions: |
-  If you came across any indications that a bounty was paid out for this
-  vulnerability, fill it out here. Or correct it if the information already here
-  was wrong. Otherwise, leave it blank.
-bounty:
-  amt:
-  announced:
-  url:
-reviews: []
-bugs_instructions: |
-  What bugs are involved in this vulnerability?
-
-  Please list bug IDs to https://bugzilla.kernel.org/
+CWE:
+  - 191
+ipc:
+  note: This vulnerability caused the system to crash causing a denial of service. It is less of a direct feature that is an interprocess communication more so that any ipc would be killed by the crash.
+  answer: true
+  question: |
+    Did the feature that this vulnerability affected use inter-process
+    communication? IPC includes OS signals, pipes, stdin/stdout, message
+    passing, and clipboard. Writing to files that another program in this
+    software system reads is another form of IPC.
 
-  Bug ID's can appear in several places:
-    * Mentioned in commit messages
-    * Mentioned in mailing list discussions
-    * References from NVD entry
-    * Various other places
+    Answer must be true or false.
+    Write a note about how you came to the conclusions you did, regardless of
+    what your answer was.
+CVSS:
 bugs: []
-fixes_instructions: |
-  Please put the commit hash in "commit" below.
+i18n:
+  note: Numbers as far as I know don't have much in the way of internationalization issues. It is really just math.
```

Comment:
This answer is giving, "no, I guess. It's just math" Answer like you're explain this to a sibling that has no idea what this is.

Reply:
Ok, I'll add more information and look into the surrounding system again.